### DISCUSSION TITLE
What is the insight gathered from visualize_topics() bubble chart visualization?

### DISCUSSION THREAD

**Discussion by gnanukoth** [MaartenGr/BERTopic#2129] [Q&A] [answered]
Hello,
I would like to understand the visualize_topics() function better. What is the inference that I can gain from the bubble chart visualization of the topics? 
From the title it seems like it is the distance between the topics (Inter Topic Distance), implying the similarity between topics. But how does it explain the 2d values obtained from the UMAP reduction?
Is there any other way for visualizing the topics to show a more insightful visualization?
Thanks!

**Comment by MaartenGr** [comment, marked as answer]
The points that you see are the reduced topic embeddings. The topic embeddings are the average of all document embeddings within a topic. The topic embeddings are then reduced in their dimensionality using UMAP in order to keep some of its global and local structure in high-dimensional space. Since UMAP aims to keep these distances in tact, this was called the inter-topic distances.

**Reply by gnanukoth** [reply]
Understood @MaartenGr. So with respect to this, how can the overlap and distance between bubbles be understood? Can it be read as the overlapping bubbles are similar in content/topic while the ones farther apart are not?

**Reply by MaartenGr** [reply]
The sizes of the bubbles merely represent the size of the topic itself and should not be used to interpret the relative distance itself.
